packages/core/networks/1_args.json

```diff
@@ -0,0 +1,8 @@
+{
+    "0x38015ddB8b34c84934Cff058F571349Cc7d4139d": [],
+    "0x9A077D4fCf7B26a0514Baa4cff0B481e9c35CE87": [
+        "0x40f941E48A552bF496B154Af6bf55725f18D77c3",
+        "0x7c96d6235CfaaCcAc5d80fCe74E6032B25dd1F03",
+        "0x0000000000000000000000000000000000000000"
+    ]
+}
```

Comment:
what does this file do?

Reply:
To verify the contract, we have to tell etherscan what args we passed to it on construction. This file stores that so when we programmatically upload, that data is available.